data_dic.R

```diff
@@ -0,0 +1,78 @@
+library("RSQLite")
+
+con <- dbConnect(SQLite(), dbname = "~/Dropbox/Bergen/seedclimComm/database/seedclim.sqlite")
+
+turfs <- tbl(con, "turfs") %>% collect()
+plots <- tbl(con, "plots") %>% collect()
+blocks <- tbl(con, "blocks") %>% collect() %>% 
+  select(blockID, siteID)
+sites <- tbl(con, "sites") %>% select(siteID, siteCode, latitude, longitude, elevation = `altitude(DEM)`, annualPrecipitation_gridded, temperature_level, summerTemperature_gridded, precipitation_level, geology) %>% collect() 
+
+
+subTurfCommunity <- tbl(con, "subTurfCommunity") %>% collect()
+turfCommunity <- tbl(con, "turfCommunity") %>% collect()
+
+subTurfEnvironment <- tbl(con, "subTurfEnvironment") %>% collect()
+turfEnvironment <- tbl(con, "turfEnvironment") %>% collect()
+
+taxon <- tbl(con, "taxon") %>% collect()
+site_attributes <- tbl(con, "site_attributes") %>% collect()
+character_traits <- tbl(con, "character_traits") %>% collect()
+numeric_traits <- tbl(con, "numeric_traits") %>% collect()
+
+
+
+### SITE
+min <- map_df(sites, min) %>% 
```

Comment:
not entirely sure what you are doing here - but it looks to be an inefficient way to do it. Maybe you want
`sites %>% summarise(across(where(is.numeric), list(min = min, max = max))` or `sites %>% summarise(across(where(is.numeric), ~glue::glue("{min(.)}-{max(.)}")))` or `sites %>% pivot_longer(where(is.numeric)) %>% group_by(name) %>% summarise(range = "glue::glue("{min(value)}-{max(value)}")")`